tests/test_calexpCutout.py

```diff
@@ -0,0 +1,145 @@
+#
+# LSST Data Management System
+# Copyright 2008-2016 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+"""Test configuration of CalexpCutoutTask and the run method.
+
+Run the task on one test image and perform various checks on the results
+"""
+import os
+import random
+import unittest
+from astropy.coordinates import SkyCoord
+from astropy.table import QTable
+import astropy.units as u
+
+import lsst.utils
+import lsst.utils.tests
+from lsst.afw.image import ExposureF
+from lsst.pipe.tasks.calexpCutout import CalexpCutoutTask
+
+random.seed(208241138)
+
+packageDir = lsst.utils.getPackageDir('pipe_tasks')
+datadir = os.path.join(packageDir, 'tests', "data")
+
+
+def make_data(bbox, wcs, border=100, ngood=13, nbad=1):
```

Comment:
Why no an nedge parameter? Not a big deal but it is a surprising omission.

Reply:
I was lazy and didn't want to figure out how to sample random points on the edge.  I can add it.

Reply:
Eh, no need. Not that it should be hard, but...not very important.